src/lib.d

```diff
@@ -10,44 +10,47 @@ module ddmd.lib;
 
 import ddmd.globals;
 
-static if (TARGET_WINDOS)
+// Include all of these once these modules have been ported to all platforms.
+version (Windows)
 {
     import ddmd.libomf;
     import ddmd.libmscoff;
 }
-else static if (TARGET_LINUX || TARGET_FREEBSD || TARGET_OPENBSD || TARGET_SOLARIS)
+else version (OSX)
 {
-    import ddmd.libelf;
+    import ddmd.libmach;
 }
-else static if (TARGET_OSX)
+else version (Posix)
 {
-    import ddmd.libmach;
+    import ddmd.libelf;
 }
 else
 {
-    static assert(0, "unsupported system");
+    static assert(0, "OS not supported.");
 }
 
 extern (C++) class Library
 {
 public:
     static Library factory()
     {
-        static if (TARGET_WINDOS)
+        // This should be turned into runtime logic based on TARGET_* once
+        // these library generation modules are made cross-platform.
+        version (Windows)
         {
             return (global.params.mscoff || global.params.is64bit) ? LibMSCoff_factory() : LibOMF_factory();
         }
-        else static if (TARGET_LINUX || TARGET_FREEBSD || TARGET_OPENBSD || TARGET_SOLARIS)
```

Comment:
>This change will never enable cross-compilation. Other changes, that are not likely to be helped by this change, will do that.

If by "this change" you mean the modifications to this file, I already granted that several times, as it's a hack to avoid going much farther towards cross-compilation.  If you mean "this whole pull," which is clearly what my quote above referred to when I quoted you above, you're wrong.

>Not true.

Then please specify exactly which OS was used differently than it is now.

>No, I would like you to replace this change with something that adds value.

I don't know what adds value for you.  If you simply mean expanding `version (Posix)` out to every OS, that's what was done before, so at best it's keeping things the same.  If it's something else, you'll have to specify what you mean.  These repeated cryptic review comments just waste both our time, when you can specify what you mean in the first place, rather than just saying "Not true" or "Not an improvement," with no explanation given.